packages/remix-server-runtime/server.ts

```diff
@@ -160,13 +178,15 @@ async function handleDataRequestRR(
     }
 
     if (isRouteErrorResponse(error)) {
-      logServerErrorIfNotAborted(error.error || error, request, serverMode);
+      if (error.error) {
+        handleError(error.error);
+      }
```

Comment:
At this point these should always have an `error` prop since they should all be internal-router-generated errors.  User-thrown Responses would have been returned as a plain Response above.  But just to be safe, we don't want user-thrown Responses logging